components/stage/scripts/start.py

```diff
@@ -5,12 +5,14 @@
 from cloudify import ctx
 
 ctx.download_resource(
-    join('components', 'utils.py'),
-    join(dirname(__file__), 'utils.py'))
+        join('components', 'utils.py'),
+        join(dirname(__file__), 'utils.py'))
 import utils  # NOQA
 
 STAGE_SERVICE_NAME = 'stage'
 
-
-ctx.logger.info('Starting Stage (UI) Service...')
-utils.start_service(STAGE_SERVICE_NAME)
+ignore_ui = ctx.instance.runtime_properties['ignore_ui']
+print "ignore_ui={0}".format(ignore_ui)
+if ignore_ui != 'True':
```

Comment:
skip_stage_installation = 'skip_stage_installation' in ctx.instance.runtime_properties

if skip_stage_installation:
    return